internal/kubernetes/operator/config_exporter.go

```diff
@@ -395,3 +402,33 @@ func (e *ConfigExporter) exportAtlasStreamProcessing(projectName string) ([]runt
 
 	return result, nil
 }
+
+func (e *ConfigExporter) exportAtlasFederatedAuth(projectName string) ([]runtime.Object, error) {
+	result := make([]runtime.Object, 0)
+	// Gets the FederationAuthSetting
+	federatedAuthentificationSetting, err := e.dataProvider.FederationSetting(&admin.GetFederationSettingsApiParams{OrgId: e.orgID})
+	if err != nil {
+		return nil, err
+	}
+	// Does not have an IdenityProvider set then no need to generate
+	if federatedAuthentificationSetting.IdentityProviderStatus == nil || *federatedAuthentificationSetting.IdentityProviderStatus == "INACTIVE" {
```

Comment:
https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_5_best_practices.md#using-getters-instead-of-direct-field-access

Reply:
Thank you! I really do not know why I did not make use of those! Much simpler now.